README.md

```diff
@@ -324,10 +324,50 @@ var path = await PdftronFlutter.saveDocument();
 Return a map object with values for position (bottom-left: `x1`, `y1`; top-right: `x2`, `y2`) and size (`width`, `height`) of the crop box for specified page. Values all have type `double`.
 
 ```dart
-var cropBox = await controller.getPageCropBox(1);
+var cropBox = await PdftronFlutter.getPageCropBox(1);
 print('The width of crop box for page 1 is: ' + cropBox.width.toString());
 ```
 
+### PdftronFlutter.setToolMode(String)
```

Comment:
`setToolMode` or simply `setTool`? @sgong-pdftron does Android speak in terms of tool "modes", because on iOS we simply have tools.

Reply:
@JamesTron I believe this is to have consistent naming with React Native, not related to android... either work for me

Reply:
Ok....